internal/server/data/access_key.go

```diff
@@ -156,11 +156,6 @@ func ValidateAccessKey(tx GormTxn, authnKey string) (*models.AccessKey, error) {
 		}
 
 		t.ExtensionDeadline = time.Now().UTC().Add(t.Extension)
-
-		// Set the orgID in the tx. This is only necessary because our data
-		// layer requires an orgID be set in the transaction. If we remove
-		// that requirement, we can remove this line as well.
-		tx = NewTransaction(tx.GormDB(), t.OrganizationID)
```

Comment:
Does this mean that we are saving this access key without an org ID set or is that in the context at this point?